### DISCUSSION TITLE
how to know keycode

### DISCUSSION THREAD

**Discussion by zzx41375004** [xxxily/h5player#246] [General]
hello!
i want to change some shortcuts, but i don't know the keycode for some keys, for example '-' or '+'.
please help

你好。
我想改一些快捷键，例如用‘-’和‘+’来控制倍速
但如何知晓其对应的keycode？
望指教，谢谢！

**Comment by JAGulin** [comment]
Try this: http://keycode.info/
I found as of now no way to change shortcuts apart from changing the actual plugin code.

**Reply by zzx41375004** [reply]
thank you for help!
i found that the language of this plugin is javascript, so it's not hard to make some small modifications

**Comment by onionycs** [comment]
题主最后更改成功了吗，可以教教我吗？尴尬😓

**Comment by onionycs** [comment]
我想绑定F键为全屏，怎么做呢
